pkg/manifest/manifest_writer.go

```diff
@@ -23,12 +23,16 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-type ManifestWriterContext struct {
-	Fs           afero.Fs
+// WriterContext holds all information for [WriteManifest]
+type WriterContext struct {
+	// Fs holds the abstraction of the file system.
+	Fs afero.Fs
+
+	// ManifestPath holds the path from where the manifest should be loaded.
```

Comment:
is that really the path a manifest was loaded from, for **Writer** or is it where the manifest will be written to?

Reply:
good catch :d